source/src/main/java/com/trustpilot/connector/dynamodb/DynamoDBSourceConnectorConfig.java

```diff
@@ -204,12 +209,21 @@ public static ConfigDef baseConfigDef() {
 						ConfigDef.Width.MEDIUM,
 						DST_TOPIC_PREFIX_DISPLAY)
 
+				.define(SRC_INIT_SYNC_SKIP_CONFIG,
+						ConfigDef.Type.BOOLEAN,
+						SRC_INIT_SYNC_SKIP_DEFAULT,
+						ConfigDef.Importance.LOW,
+						SRC_INIT_SYNC_SKIP_DOC,
+						CONNECTOR_GROUP, 2,
+						ConfigDef.Width.MEDIUM,
+						SRC_INIT_SYNC_SKIP_DISPLAY)
+
 				.define(SRC_INIT_SYNC_DELAY_CONFIG,
 						ConfigDef.Type.INT,
 						SRC_INIT_SYNC_DELAY_DEFAULT,
 						ConfigDef.Importance.LOW,
 						SRC_INIT_SYNC_DELAY_DOC,
-						CONNECTOR_GROUP, 2,
+						CONNECTOR_GROUP, 3,
```

Comment:
What is 'CONNECTOR_GROUP' ?

Reply:
This class is used to display configs in the UI on something like Confluent Hub (width, grouping, importance). No real use outside of that.